includes/wikia/Extensions.php

```diff
@@ -1394,7 +1394,7 @@
 // SER-3444 always use GCS
 $wgEnabledFileBackend = 'gcs-backend';
 
-
+include( "{$IP}/extensions/wikia/CoppaTool/legacy/CoppaToolLegacyTasks.setup.php" );
```

Comment:
I'm fine with this in app, just add a comment that it's required to be enabled on all wikis because these tasks need to be run regardless for Special:CoppaTool being enabled or not.